src/Microsoft.Azure.AppConfiguration.IsolatedAzureFunctions/AzureAppConfigurationRefreshExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration;
+using Microsoft.Extensions.Hosting;
+using System;
+using System.Linq;
+
+namespace Microsoft.Azure.AppConfiguration.IsolatedAzureFunctions
+{
+    /// <summary>
+    /// Extension methods for Azure App Configuration.
+    /// </summary>
+    public static class AzureAppConfigurationRefreshExtensions
+    {
+        /// <summary>
+        /// Configures a middleware for Azure App Configuration to use activity-based refresh for data configured in the provider.
+        /// </summary>
+        /// <param name="builder">An instance of <see cref="IFunctionsWorkerApplicationBuilder"/></param>
+        public static IFunctionsWorkerApplicationBuilder UseAzureAppConfiguration(this IFunctionsWorkerApplicationBuilder builder)
+        {
+            if (builder == null)
```

Comment:
Considering the usage of an extension method (`builder.UseAzureAppConfiguration`) letting the nullref happen in this case is more expected.